portal/portal-impl/impl/src/bundle/sitenav.properties

```diff
@@ -110,8 +110,10 @@ moresite_other = OTHER
 moresite_unknown_term = (unknown academic term)
 moresite_organize_favorites = Organize Favorites
 moresite_favorites = Favorites
-moresite_add_to_favorites = Add [site] to favorites
-moresite_remove_from_favorites = Remove [site] from favorites
+moresite_add_to_favorites = Add {0} to favorites
+moresite_remove_from_favorites = Remove {0} from favorites
```

Comment:
This wasn't working properly for non english languages.
Remaining files should be revised to show the proper site name.